### PR TITLE
Updated Contact Page

### PR DESCRIPTION
**Pull request by anthonyy3995**
Created basic layout of Contact Page. Incomplete, there is still some further spacing/dimensioning to add.

### CODE REVIEW

File: src/pages/Contact.css [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    justify-content: center;
+
+}
+.title{
+    /* style the CONTACT US */
+    width: 468px;
+    height: 68px;
+    /* margin-left: 468px; */
+    margin-left: 242px;
+    top: 159px;
```
Comment (TheWillMundy) [inline src/pages/Contact.css:13]: Usually we don't want to put these kinds of constraints in, as it makes the webpage less responsive; instead, let's just let Flexbox handle it (the earlier declaration in `.titleContainer` of `display: flex ... justify-content: center` will center this automatically)

File: src/pages/Contact.css [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    flex-direction: row;
+    justify-content: space-evenly;
+
+}
+.messageContainer{
+    display: flex;
+    flex-direction: column;
+
+    /* why isn't this working? */
+    align-content: space-between;
```
Comment (TheWillMundy) [inline src/pages/Contact.css:34]: Should be `align-items` not `align-content`

File: src/pages/Contact.css [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+    justify-content: center;
+}
+.messageTitle{
+
+     width: 495px;
+     height: 42px;
+     margin-left: 162px;
```
Comment (TheWillMundy) [inline src/pages/Contact.css:42]: As noted above

File: src/pages/Contact.css [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+    justify-content: center;
+}
+.messageTitle{
+
+     width: 495px;
+     height: 42px;
+     margin-left: 162px; 
+
+
+     text-align: center;
```
Comment (TheWillMundy) [inline src/pages/Contact.css:45]: Move this into the `.messageContainer` declaration

File: src/pages/Contact.css [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+
+     font-style: normal;
+     font-weight: 600;
+     font-size: 30px;
+}
+.smallInput{
+    /* style the inputs */
+    margin-left: 116px;
+    height: 52px;
+    width: 608px;
```
Comment (TheWillMundy) [inline src/pages/Contact.css:55]: As noted above; let flexbox handle it

File: src/pages/Contact.css [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+
+}
+.largeInput{
+    /* style the inputs */
+    margin-left: 116px;
+    height: 288px;
+    width: 608px;
```
Comment (TheWillMundy) [inline src/pages/Contact.css:62]: Ditto ^

File: src/pages/Contact.css [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+
+}
+.followContainer{
+    display: flex;
+    flex-direction: column;
+    align-content: space-between;
```
Comment (TheWillMundy) [inline src/pages/Contact.css:68]: As noted above; change to `align-items`

File: src/pages/Contact.css [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+
+}
+.followTitle{
+    width: 495px;
+    height: 42px;
```
Comment (TheWillMundy) [inline src/pages/Contact.css:73]: As noted above

File: src/pages/Contact.css [hunk 9]
```diff
@@ -0,0 +1,138 @@
...
+
+    
+
+    text-align: center;
```
Comment (TheWillMundy) [inline src/pages/Contact.css:77]: Move to `.followContainer`

File: src/pages/Contact.css (hunk 7 above)
Comment (TheWillMundy) [reply src/pages/Contact.css:68]: Also change to `align-items: center`

File: src/pages/Contact.css [hunk 10]
```diff
@@ -0,0 +1,138 @@
...
+    font-weight: 600;
+    font-size: 30px;
+    line-height: 42px;
+
+}
+.logoContainer{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between; 
+    /* how to space the buttons correctly? */
```
Comment (TheWillMundy) [inline src/pages/Contact.css:90]: Give the `.logoContainer` a width of 50%, and with the above change to `align-items: center` in the `.followContainer` it should fix things

File: src/pages/Contact.css [hunk 11]
```diff
@@ -0,0 +1,138 @@
...
+    height: 52px;
+    /* margin-left: 1137px; */
+
+    border-radius: 10px;
+}
+
+.newsletterTitle{
+
+    width: 495px;
+    height: 42px;
```
Comment (TheWillMundy) [inline src/pages/Contact.css:121]: Remove

File: src/pages/Contact.css [hunk 12]
```diff
@@ -0,0 +1,138 @@
...
+
+    font-style: normal;
+    font-family: PingFang SC;
+    font-size: 30px;
+    text-align: center;
```
Comment (TheWillMundy) [inline src/pages/Contact.css:126]: Remove

File: src/pages/Contact.js [hunk 13]
```diff
@@ -1,8 +1,46 @@
...
+                    CONTACT US
+                </h1>
+            </div>
+            <div className = "bottomContainer">
+                <div className = "messageContainer">
+                    <h2 className = "messageTitle">
+                        MESSAGE
+                    </h2>
+                    <input className = "smallInput" />
+                    <input className = "smallInput" style={{top:"472px"}}/>
```
Comment (TheWillMundy) [inline src/pages/Contact.js:18]: Remove the `style={{...}}`

File: src/pages/Contact.css [hunk 14]
```diff
@@ -0,0 +1,138 @@
...
+    justify-content: space-evenly;
+
+}
+.messageContainer{
+    display: flex;
+    flex-direction: column;
+
+    /* why isn't this working? */
+    align-content: space-between;
+
```
Comment (TheWillMundy) [inline src/pages/Contact.css:35]: Add a `height: ...` to the container, and change `justify-content: center` to `justify-content: space-between` while making `align-items: center`

**Review by TheWillMundy — CHANGES_REQUESTED** [review]
Great work so far! Most of the comments are focused on removing "absolute positioning" (so taking out the parts where you assigned a `height` and `width` to each element, as well as `margin-left` or `margin-right` attributes); I also gave some tips on how to fix some of the spacing issues you were facing.

File: src/pages/Contact.css (hunk 5 above)
Comment (anthonyy3995) [reply src/pages/Contact.css:55]: If I get rid of these dimensions though, the input objects shrink. 
Screen Shot 2020-11-18 at 8 13 55 PM

**Comment by TheWillMundy** [comment]
Ported your CSS into the TailwindCSS-based version and applied very very minor fixes; overall, great work!

**Pull request closed without merging**